### DISCUSSION TITLE
Where is the database?

### DISCUSSION THREAD

**Discussion by WolfgangFahl** [ciur/papermerge#632] [General]
**Description**

As asked in https://www.reddit.com/r/Papermerge/comments/1ez7n06/papermount_making_inbox_and_db_available_via_mount/ i would love to get access to the database
**Info:**
 - Papermerge Version 3.0.3 using https://wiki.bitplan.com/index.php/Papermerge#Script_to_manage_papermerge_docker_installation
 
 when i bash into the docker container 
 ```bash
root@39f667b41078:/core_app# find . -name *.sqlite3
```

does not show anything despite the info i found that sqlite is the default storage. The peculiar thing is that when i did my first installation attempt with:
```bash
#!/bin/bash
# WF 2024-08-23
# start papermerge
docker run -p 8008:8000 \
    -e PAPERMERGE__MAIN__SECRET_KEY=abc \
    -e DJANGO_SUPERUSER_PASSWORD=123 \
    -e DJANGO_SUPERUSER_USERNAME=john \
    papermerge/papermerge:latest
```

then there was an sqlite3 database. I wonder if papermerge:latest is not a misnomer since 3.0.3 seems to be the most current one.

**Comment by ciur** [comment]
While you are inside docker container just type `env` command, it will print all current environment variables. Among environment variables you are interested in those starting with `PAPERMERGE__` prefix; in particular information about database (if it is the sqlite it will be the absolute path to sqlite file) is in `PAPERMERGE__DATABASE__URL` environment variable:

https://docs.papermerge.io/3.2/settings/database/

Default sqlite file path is [/db/db.sqlite3](https://github.com/papermerge/papermerge-core/blob/master/docker/prod/config/settings.py#L24)

**Comment by WolfgangFahl** [comment]
indeed
```bash
papermerge -b
root@ad0b25bd069a:/core_app# ls -l /db
total 240
-rw-r--r-- 1 root root 241664 Aug 24 13:14 db.sqlite3
```
